CHANGELOG.rst

```diff
@@ -20,6 +20,8 @@ number of the code change for that issue.  These PRs can be viewed at:
 3.xx.x (unreleased)
 ===================
 
+- Fixed metawcs ecsv filename issue and lingering files. [#2073]
```

Comment:
Is it more accurate to say `and removed lingering files`?

Reply:
fixed with [142af7b61fe98c6f0b0e9315d79672ba03001ddd](https://github.com/spacetelescope/drizzlepac/pull/2073/commits/142af7b61fe98c6f0b0e9315d79672ba03001ddd)